src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesBackupProtectionPolicy.md

```diff
@@ -67,15 +67,15 @@ This command gets the protection policy named DefaultPolicy, and then stores it
 
 ### -BackupManagementType
 Specifies the Backup management type.
-Currently, only AzureVM, AzureStorage is supported.
+Currently, only AzureVM, AzureStorage, AzureWorkload is supported.
 
 ```yaml
 Type: System.Nullable`1[Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets.Models.BackupManagementType]
 Parameter Sets: WorkloadBackupManagementTypeParamSet
 Aliases:
-Accepted values: AzureVM, MARS, SCDPM, AzureBackupServer, AzureSQL, AzureStorage, AzureWorkload
+Accepted values: AzureVM, MARS, SCDPM, AzureBackupServer, AzureSQL, AzureStorage, AzureWorkload, MAB
 
-Required: True
+Required: true
```

Comment:
why is this changed?

Reply:
I have reverted this to the previous text, as this is no longer needed.

Reply:
resolved